acme_diags/driver/qbo_driver.py

```diff
@@ -8,6 +8,22 @@
 from acme_diags.derivations import acme, default_regions
 from acme_diags.driver import utils
 from acme_diags.plot.cartopy.qbo_plot import plot
+import MV2
+
+def unify_plev(var):
+    """
+    Given a data set with a z-axis, 
+    convert to the plev with units: hPa and make sure z is positive up (same as model data)
+    """
+    var_plv = var.getLevel()
+    if var_plv.units == 'Pa':
+        var_plv[:] = var_plv[:]/100.0 #convert Pa to mb
+        var_plv.units = 'hPa'
+        var.setAxis(1, var_plv)
+    # For positive down plevel: from TOP (level 0) to BOTTOM (last level), i.e Plev value
+    # going up with each level", revers it. 
```

Comment:
I think a better wording might be `if z is positive down (level 0 < last level), then reverse z`.

Reply:
I changed the wording to be more clear..